src/aura/BGE_ConfigurationWizard/BGE_ConfigurationWizard.cmp

```diff
@@ -338,76 +339,94 @@
+                                                      variant="base"
+                                                      class="bgeLinkButton"
+                                                      onclick="{!c.handleProcessingSettingsChange}" />
                                 </lightning:layoutItem>
+
+                                <!-- BEGIN ADVANCED OPTIONS FOR STEP 5 -->
+                                <aura:if isTrue="{!v.batchMetadata.showAdvancedOptions}">
+                                    <lightning:layoutItem size="12">
+                                        <lightning:layout horizontalAlign="spread" multipleRows="true">
+                                            <lightning:layoutItem class="slds-p-around_x-small" size="12">
+                                                <label class="slds-form-element__label" for="donationDateRange">
+                                                    <abbr class="slds-required" title="required">*</abbr>
+                                                    {!v.batchMetadata.labels.donationDateRangeLabel}
+                                                </label>
+                                                <lightning:helptext content="{!$Label.c.bgeBatchDonationDateRangeHelp}" />
+                                                <lightning:input aura:id="donationDateRange"
+                                                                    label="{!v.batchMetadata.labels.donationDateRangeLabel}"
+                                                                    disabled="{!or(v.batchInfo.donationMatchingBehavior == v.batchMetadata.labels.DoNotMatch, v.batchInfo.noMatchOnDate)}"
+                                                                    type="number" required="true" variant="label-hidden"
+                                                                    value="{!v.batchInfo.donationDateRange}"/>
+                                            </lightning:layoutItem>
+                                            <lightning:layoutItem class="slds-p-around_x-small" size="12">
+                                                <label class="slds-form-element__label" for="donationMatchingClass">
+                                                    {!v.batchMetadata.labels.donationMatchingClassLabel}
+                                                </label>
+                                                <lightning:helptext content="{!$Label.c.bgeBatchDonationMatchingClassHelp}" />
+                                                <lightning:input aura:id="donationMatchingClass"
+                                                                label="{!v.batchMetadata.labels.donationMatchingClassLabel}"
+                                                                disabled="{!v.batchInfo.donationMatchingBehavior == v.batchMetadata.labels.DoNotMatch}"                                                                
```

Comment:
I think we've already discussed, but `v.batchMetadata.labels.DoNotMatch` is coming from the service class and not from an actual label, correct?

Reply:
Yes, exactly